VotingApplication/VotingApplication.Web.Api/Scripts/Create.js

```diff
@@ -33,19 +33,19 @@
                 contentType: 'application/json',
 
                 data: JSON.stringify({
-                    Name: self.pollName(),
-                    Creator: self.creatorName(),
-                    Email: self.creatorEmail(),
-                    TemplateId: self.templateId(),
-                    VotingStrategy: self.strategy(),
-                    MaxPoints: self.maxPoints(),
-                    MaxPerVote: self.maxPerVote(),
-                    InviteOnly: self.inviteOnly(),
-                    AnonymousVoting: self.anonymousVoting(),
-                    RequireAuth: self.requireAuth(),
-                    Expires: self.expiry(),
-                    ExpiryDate: new Date(self.expiryDate()),
-                    OptionAdding: self.optionAdding()
+                    Name: self.pollName() || undefined,
```

Comment:
It's how the new validation works. Fields that are sent by javascript but with a defined value (even null) can be considered invalid input. Setting the value to undefined is equivalent to not setting the value at all, which is considered valid by the validator. (provided it isn't flagged as a required field)